Source/WebCore/rendering/RenderAttachment.cpp

```diff
@@ -43,10 +43,18 @@ namespace WebCore {
 
 using namespace HTMLNames;
 
+// Co-dependent with CSS #attachment-container width and height.
+#if (defined(WTF_PLATFORM_IOS_FAMILY) && WTF_PLATFORM_IOS_FAMILY)
+static constexpr IntSize attachmentWideLayoutIntrisicSize { 338, 92 };
+#else
+static constexpr IntSize attachmentWideLayoutIntrisicSize { 266, 80 };
```

Comment:
Do we really need these, or can we pull from the `RenderStyle`?

Reply:
`RenderStyle` didn't have these values, so instead I'm using the size of the laid-out shadow tree.